modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectDefinitionTestUtil.java

```diff
@@ -30,7 +29,7 @@
  */
 public class ObjectDefinitionTestUtil {
 
-	public static ObjectDefinition addCustomObjectDefinition()
+	public static ObjectDefinition addAndPublishCustomObjectDefinition()
```

Comment:
@p-albuquerque I think it can be `publishCustomObjectDefinition` or just `publishObjectDefinition`

Reply:
use the same name from the local service method